README.md

```diff
@@ -199,6 +199,7 @@ Assign a default value for each setting by passing environment variables to Libr
 | `USE_HLS`               | `["on", "off"]`                                                                                     | `off`         |
 | `HIDE_HLS_NOTIFICATION` | `["on", "off"]`                                                                                     | `off`         |
 | `AUTOPLAY_VIDEOS`       | `["on", "off"]`                                                                                     | `off`         |
+| `VISIT_REDDIT_CONFIRMATION`     | `["on", "off"]`                                                                                     | `off`         |
```

Comment:
Since this is now a negative, we should change the name of this variable to `DISABLE_VISIT_REDDIT_CONFIRMATION` to make it clear that this variable being set to `on` now **disables** the warning popup.